workflow/scripts/create_somalier_mqc_config.py

```
@@ -0,0 +1,480 @@
+        return 0.0
+
+    # Default to unrelated
+    return 0.0
+
+
+try:
+    args = parse_args()
+
+    # Load somalier MultiQC config
+    with open(args.config, 'r') as report_configs:
+        somalier_mqc_configs = yaml.load(report_configs, Loader=yaml.FullLoader)
```

Ruff (0.15.13)

[error] Probable use of unsafe loader `FullLoader` with `yaml.load`. Allows instantiation of arbitrary objects. Consider `yaml.safe_load`.

(S506)